sql/topology/nodeNetwork.sql

```diff
@@ -31,9 +31,9 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 
 --v2.6
-CREATE FUNCTION pgr_nodeNetwork(
-  TEXT, -- edge table (required)
-  DOUBLE PRECISION, -- tolerance (required)
+CREATE OR REPLACE FUNCTION pgr_nodeNetwork(
```

Comment:
This should be `CREATE FUNCTION pgr_nodeNetwork(` with out the replace as the extension creates functions.